### PR TITLE
Creating database

### PR DESCRIPTION
**Pull request by seth-at-at**

### CODE REVIEW

**Comment by seth-at-at** [comment]
Took out users, switched it to diary.

File: lib/models/diary.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+function find(id){
+  return database.raw('SELECT * FROM diary WHERE id=?', [id])
+}
+
+function findName(name){
+  return database.raw('SELECT * FROM diary WHERE name=?', [name])
+}
+
+module.exports = {
+  create: createDiary,
```
Comment (tmikeschu) [inline lib/models/diary.js:30]: These names should match 👍  That also allows you to do:
```js
function matchingName () {}
function otherMatchingName () {}

module.exports = {
  matchingName,
  otherMatchingName
}
```

File: db/migrations/20170629134831_create-users-table.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (tmikeschu) [inline db/migrations/20170629134831_create-users-table.js:1]: Users are not required for the project. What's the call here?

File: lib/models/diary.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+
+function createDiary(name){
```
Comment (tmikeschu) [inline lib/models/diary.js:6]: You shouldn't need to make multiple diaries...

**Review by tmikeschu — COMMENTED** [review]
I see you asked for a review, but it's more helpful if you ask specific questions. It looks like you're taking some interesting and unnecessary database decisions, see comments.

**Pull request merged**